views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
```

Comment:
I wouldn't use the `<style>` tag, I would place inside the public => stylesheets => style.css as this is available through your whole application